tock/api/urls.py

```diff
@@ -14,6 +14,11 @@
         ReportingPeriodAudit.as_view(),
         name='ReportingPeriodAudit'
     ),
+    re_path(
+        r'^submissions/(?P<num_past_reporting_periods>\d+).json$',
```

Comment:
Opportunity here to borrow the approach being used just above this for the ProjectInstanceView and avoid needing regular expressions. We'll also be able to skip casting the inbound value to an integer ourselves in the view.
```suggestion
    path(
        'submissions/<int:num_past_reporting_periods>.json',
```

Reply:
This is way better, thanks!